modules/http.lua

```diff
@@ -222,9 +223,21 @@ function ClientRequest:initialize(options, callback)
     if cfound then
       connection_found = value
     end
+    local uafound = key:lower() == 'user-agent'
+    if uafound then
+      user_agent_found = value
+    end
     table.insert(self, header)
   end
 
+  if not user_agent then
+    user_agent = 'luvit/http/' .. exports.version .. ' luvi/' .. luvi.version
```

Comment:
Should calculate the default value only once outside the closure.  It's not going to change for the lifetime of the process.